src/vs/editor/browser/viewParts/linesDecorations/linesDecorations.ts

```diff
@@ -97,7 +97,7 @@ export class LinesDecorationsOverlay extends DedupOverlay {
 			const classNames = toRender[lineIndex];
 			let lineOutput = '';
 			for (let i = 0, len = classNames.length; i < len; i++) {
-				lineOutput += '<div class="cldr ' + classNames[i] + common;
+				lineOutput += '<div class="cldr codicon ' + classNames[i] + common;
```

Comment:
I don't think it would be good to make all the lines decorations be codicons. I think the "codicon" class name should move and become part of each decoration, as necessary.